.github/workflows/main.yml

```diff
@@ -17,4 +17,4 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v1
     - name: install and then run linter
-      run: npm install eslint && npm run lint
\ No newline at end of file
+      run: npm install eslint && npm run lint && npm run lint:fix
```

Comment:
&& will not work in this case.
since, if npm run lint has some errors it will stop the other commands the instance an error is encountered. `||` may work.

Secondly lint:fix will not work, since it will only fix the code in the ci and the changes will not be reflected in the the main code unless it is pushed to your branch.

Reply:
Yaa thanks I realized that. Removed those changes and fixed the linting using above command.
@Abhishek-kumar09 you can review it now :)